src/pkg/clouds/aws/ecs/cfn/setup.go

```diff
@@ -70,6 +71,19 @@ func (a *AwsEcs) updateStackAndWait(ctx context.Context, templateBody string) er
 		return err
 	}
 
+	// Check the template version first, to avoid updating to an outdated template; TODO: can we use StackPolicy/Conditions instead?
+	if dso, err := cfn.DescribeStacks(ctx, &cloudformation.DescribeStacksInput{StackName: &a.stackName}); err == nil && len(dso.Stacks) == 1 {
+		for _, output := range dso.Stacks[0].Outputs {
+			if *output.OutputKey == outputs.TemplateVersion {
+				deployedRev, _ := strconv.Atoi(*output.OutputValue)
+				if deployedRev > TemplateRevision {
+					fmt.Println("CloudFormation stack", a.stackName, "is newer than the current template; skipping update")
```

Comment:
I think we may want to make it a more verbose failure and prompt for cli update

Reply:
done